src/transformers/models/zoedepth/image_processing_zoedepth.py

```diff
@@ -442,3 +449,129 @@ def preprocess(
 
         data = {"pixel_values": images}
         return BatchFeature(data=data, tensor_type=return_tensors)
+
+    def post_process_depth_estimation(
+        self,
+        outputs,
+        source_sizes: Union[TensorType, List[Tuple[int, int]]],
+        target_sizes: Optional[Union[TensorType, List[Tuple[int, int]], None]] = None,
+        outputs_flip=None,
+        remove_padding: Optional[Union[bool, None]] = None,
+        vmin_perc: Optional[float] = 1.0,
+        vmax_perc: Optional[float] = 99.0,
+        cmap: Optional[str] = "gray_r",
+        gamma_corrected: Optional[bool] = False,
+        normalize: Optional[bool] = False,
+    ) -> List[Dict]:
+        """
+        Converts the raw output of [`ZoeDepthDepthEstimatorOutput`] into final depth predictions and depth PIL images.
+        Only supports PyTorch.
+
+        Args:
+            outputs ([`ZoeDepthDepthEstimatorOutput`]):
+                Raw outputs of the model.
+            outputs_flip ([`ZoeDepthDepthEstimatorOutput`], *optional*):
+                Raw outputs of the model from flipped input (averaged out in the end).
+            source_sizes (`TensorType` or `List[Tuple[int, int]]`):
+                Tensor of shape `(batch_size, 2)` or list of tuples (`Tuple[int, int]`) containing the source size
+                (height, width) of each image in the batch before preprocessing.
+            target_sizes (`TensorType` or `List[Tuple[int, int]]`, *optional*):
+                Tensor of shape `(batch_size, 2)` or list of tuples (`Tuple[int, int]`) containing the target size
+                (height, width) of each image in the batch. If left to None, predictions will not be resized.
+            remove_padding (`bool`, *optional*):
```

Comment:
To be consistent with other control flags

```suggestion
            do_remove_padding (`bool`, *optional*):
```